scipy/io/_fortran.py

```diff
@@ -96,29 +108,46 @@ def __init__(self, filename, mode='r', header_dtype=np.uint32):
     def _read_size(self):
         return int(np.fromfile(self._fp, dtype=self._header_dtype, count=1))
 
-    def write_record(self, s):
+    def write_record(self, *items):
         """
         Write a record (including sizes) to the file.
 
         Parameters
         ----------
-        s : array_like
-           The data to write.
+        *items : array_like
+            The data arrays to write.
+
+        Notes
+        -----
+        Writes data items to a file::
+
+            write_record(a.T, b.T, c.T)
+
+            write(1) a, b, c
+
+        Note that data in multidimensional arrays is written in
+        row-major order --- to make them read correctly by Fortran
+        programs, you need to transpose the arrays yourself.
```

Comment:
perhaps add: `to make them read correctly by Fortran, you need to transpose the arrays yourself before writing them.` to emphasize where the transposing needs to happen.